### ISSUE TITLE
[Feature-Name]: Publish full test coverage report

### ISSUE THREAD

**Issue by mchelen-gov** [department-of-veterans-affairs/va.gov-team#65077]
### Describe the problem

Test coverage for the `vets-website` repo can be seen here: https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/frontend-support-dashboard/unit-test-coverage-report/

This report only shows summaries of the test coverage, and does not include specific files or lines. That makes it difficult for VFS and Platform engineers to improve test coverage.

### Who will benefit

- VFS engineers
- Platform engineers
- Product owners
- Veterans

### Describe your idea

A full test coverage report can be generated for `vets-website` with this command:
`yarn test:unit --coverage --coverage-html`

This report shows much more detail, and could be published as well.

### Provide evidence

VFS and Platform engineers have expressed uncertainty of how to improve test coverage.

### Platform Mission

- [X] Keeps these products and services available, stable, and secure.
- [X] Ensures they are consistent and meet high standards of quality in Veteran-centered user experience and technical approach.
- [X] Provides shared tools, software, support, and technical infrastructure to accelerate their development.
- [x] Other

**Comment by JoeTice** [comment]
Platform Reliability Team will take ownership of this request and will move it onto our board.

**Comment by steel36** [comment]
Status update on Requested Feature: @JoeTice Has there been any progress on this item? Is it still under consideration or active in some capacity?

**Comment by tayism** [comment]
Hi, @JoeTice, any recent updates here? Thanks!

**Comment by tayism** [comment]
Hi @JoeTice -- were there any recent updates to share on this one? Ensuring it's attached to the correct column on the board. Thanks!

**Comment by gopixelsgo** [comment]
Hi @JoeTice - did this item ever make it into Backlog or did anything become of it?

**Comment by gopixelsgo** [comment]
Hi @JoeTice - any update on this item? Thanks!

**Comment by IGallupSoCo** [comment]
After reviewing some prospects with Curt, we have agreed that I will create an overnight cron workflow that generates HTML reports for each app, which then will get uploaded to S3 with public access.

**Comment by IGallupSoCo** [comment]
Draft is up and ready for approval.  Testing will need to take place once the workflow is merged into main.  There's no risk of it disrupting CI, as it is a standalone workflow that only runs manually or once overnight.
